tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -32,7 +32,7 @@ internal static Scope CreatePlaceholderScope(Tracer tracer, NameValueHeadersColl
             tags: null,
             parent: context.SpanContext,
             serviceName: PlaceholderServiceName,
```

Comment:
```suggestion
            serviceName: PlaceholderServiceName,
```
The `PlaceholderServiceName` must be updated to what the agent expects to reject